agents/sbd/fence_sbd.py

```diff
@@ -192,6 +192,7 @@ def set_power_status(conn, options):
 
     target_status = options["--action"]
     plug = options["--plug"]
+    enable_crashdump = options["--crashdump"] == "1"
```

Comment:
This should be `enable_crashdump = "--crashdump" in options`, which supports all variants of of crashdump=1, true, yes, on and --crashdump from CLI.